src/libs/algolia/datetime/time.ts

```diff
@@ -0,0 +1,88 @@
+import { Range } from '../../typesUtils/typeHelpers'
+
+import { DATE, DAYS_IN_A_WEEK, SATURDAY_INDEX_IN_A_WEEK, SUNDAY_INDEX_IN_A_WEEK } from './date'
+
+export const MILLISECONDS_IN_A_DAY = 86400000
```

Comment:
```suggestion
export const MILLISECONDS_IN_A_DAY = 24 * 60 * 60 * 1000
```
c'est plus lisible de trouve, mais après c'est pas très important